### DISCUSSION TITLE
Week 2 Project Ideas

### DISCUSSION THREAD

**Discussion by beto0890** [USU-ITLS-x265-23-summer/class-repo#141] [General]
1.	Material to help study a second language. (This could be vocabulary learning or phrases or expressions)
2.	To do list for students or like a planner or agenda book.
3.	Any game for language learning.
4.	Website – to write a biography or to create an all about me page.
5.	Something to learn verb conjugations.
6.	Something to learn verb tenses.

**Comment by derekscholes** [comment]
I like these ideas. The main issue I forsee (at least for me it is) is the use of javascript if you want the website to be interactive. I have a difficult time creating tabs that change when clicked. If I find more information on this I will make sure I share it with you so that it will be easier to add this to your site.  It will be cool to see how this turns out, especially if you can create interactive quizzes or games to help with learning tenses or conjugations.
